library/Backends/llvm/src/Juvix/Backends/LLVM/Codegen/Block.hs

```diff
@@ -716,9 +721,22 @@ malloc size resultType = do
       callConvention
         CC.Fast
         malloc
-        (emptyArgs [Operand.ConstantOperand (C.Int Types.size_t_int size)])
+        (emptyArgs [Operand.ConstantOperand (C.Int 32 size)])
   bitCast i8Ptr resultType
 
+-- Does the sizeof code actually work!?
```

Comment:
I don't understand this comment

Reply:
shocked over `sizeOf` working, though it generates really ugly code to do its job